backend/src/main/java/com/example/backend/controller/ChecklistController.java

```diff
@@ -0,0 +1,23 @@
+package com.example.backend.controller;
+
+import com.example.backend.model.Checklist;
+import com.example.backend.service.ChecklistService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/easy-bikepackr/lists")
+public class ChecklistController {
+
+    private final ChecklistService checklistService;
+
+    @GetMapping("/")
```

Comment:
maybe "/all" so it´s more clear that you show every item. But I don't think you have to if you prefer this style